genesis_ledgers/berkeley.json

```diff
@@ -1,6 +1,6 @@
 {
   "genesis": {
-    "genesis_state_timestamp": "2023-04-06T23:30:01Z"
+    "genesis_state_timestamp": "2023-07-05T11:00:01Z"
```

Comment:
This is in UTC, could you add the UTC equivalent of 2023-07-05 12pm PT (which is 7pm I think)?)

Reply:
"2023-07-05T19:01:01Z"